README.md

```diff
@@ -403,6 +403,38 @@ end
 
 The default assertion option in 2.* was `validate_success_only=true`, but this becomes `validate_success_only=false` in 3.*. For the smoothest possible upgrade, you should set it to `false` in your test suite before upgrading to 3.*.
 
+**Test schema coverage**  
+NOTE: Currently committee only supports schema coverage for openapi schemas, and only checks coverage on responses.  
```

Comment:
Now we support tests using 'assert_schema_conform' or 'assert_response_schema_confirm' only.
(i.e. the user who use committee middleware in test )

So please write NOTICE to these user

Reply:
I have added more details to the docs